custom/conf/app.ini.sample

```diff
@@ -869,6 +869,34 @@ SHOW_FOOTER_VERSION = true
 ; Show template execution time in the footer
 SHOW_FOOTER_TEMPLATE_LOAD_TIME = true
 
+[markup.sanitizer]
+; Adding a custom sanitization rule involves modifying three shadowed
```

Comment:
This file normally contains default values only, with the occasional one-liner clarification. This explanation should be in the cheat sheet or a separate page (where you put it already).

Reply:
Since I have three keys here, I've allowed myself ~two and a bit~ three lines. ;-)

```diff
+[markup.sanitizer]
+; All three of the following keys are necessary to define one sanitation
+; policy rule and the set may be redefined multiple times to add multiple
+; rules.
+;ELEMENT = span
+;ALLOW_ATTR = class
+;REGEXP = ^\s*((math(\s+|$)|inline(\s+|$)|display(\s+|$)))+
```

I think its the minimum required to understand the section and how to use it.